tools/appveyor_install.ps1

```diff
@@ -0,0 +1,96 @@
+# Sample script to install Miniconda under Windows
```

Comment:
Just in terms of organization, we already have some Travis-specific tools in `.travis`.

I think I'd prefer to see this sort of structure:

```
integration
+- travis (existing content in .travis)
+- appveyor
+- conda
```

and leave `tools` for just developer tools as it is now.